gallery.js

```diff
@@ -1,2 +1,24 @@
 import './sass/gallery.scss'
 import '@fortawesome/fontawesome-free/css/all.css'
+
+import $ from 'jquery'
+
+//Opening modal
```

Comment:
nit: space before comment
```suggestion
// Opening modal
```